tests/suite-integration/PluginFlyvemdmAgent.php

```diff
@@ -42,7 +42,7 @@ class PluginFlyvemdmAgent extends CommonTestCase {
    protected $computerTypeId = 3;
 
    public function setUp() {
-      $this->resetState();
+      //$this->resetState();
```

Comment:
performance optimization

Reply:
you should also use the commit message type for performance